exoctk/atmospheric_retrievals/platon_wrapper.py

```diff
@@ -0,0 +1,268 @@
+"""A wrapper around the PLATON atmospheric retrieval tool.
+
+This module serves as a wrapper around the atmospheric retrieval
+software for ``platon``.  It provides methods for performing retreivals
+through multinested sampling and EMCEE methods.  For more information
+about ``platon``, please see ``https://platon.readthedocs.io``.
+
+Authors
+-------
+
+    - Matthew Bourque
+
+Use
+---
+
+    Users can perform the atmospheric retrieval by instantiating a
+    ``PlatonWrapper`` object and passing fit parameters within the
+    python environment.  An example of this is provided below
+    ::
+
+        import numpy as np
+        from platon.constants import R_sun, R_jup, M_jup
+        from exoctk.atmospheric_retrievals.platon_wrapper import PlatonWrapper
+
+        # Build dictionary of parameters you wish to fit
+        params = {
+            'Rs': 1.19,  # Required
+            'Mp': 0.73,  # Required
+            'Rp': 1.4,  # Required
+            'T': 1200,  # Required
+            'logZ': 0,  # Optional
+            'CO_ratio': 0.53,  # Optional
+            'log_cloudtop_P': 4,  # Optional
+            'log_scatt_factor': 0,  # Optional
+            'scatt_slope': 4,  # Optional
+            'error_multiple': 1,  # Optional
+            'T_star': 6091}  # Optional
+
+        # Initialize PlatonWrapper object and set the parameters
+        pw = PlatonWrapper()
+        pw.set_parameters(params)
+
+        # Add any additional fit parameters
+        R_guess = 1.4 * R_jup
+        T_guess = 1200
+        pw.fit_info.add_gaussian_fit_param('Rs', 0.02*R_sun)
+        pw.fit_info.add_gaussian_fit_param('Mp', 0.04*M_jup)
+        pw.fit_info.add_uniform_fit_param('Rp', 0.9*R_guess, 1.1*R_guess)
+        pw.fit_info.add_uniform_fit_param('T', 0.5*T_guess, 1.5*T_guess)
+        pw.fit_info.add_uniform_fit_param("log_scatt_factor", 0, 1)
+        pw.fit_info.add_uniform_fit_param("logZ", -1, 3)
+        pw.fit_info.add_uniform_fit_param("log_cloudtop_P", -0.99, 5)
+        pw.fit_info.add_uniform_fit_param("error_multiple", 0.5, 5)
+
+        # Define bins, depths, and errors
+        pw.wavelengths = 1e-6*np.array([1.119, 1.138, 1.157, 1.175, 1.194, 1.213, 1.232, 1.251, 1.270, 1.288, 1.307, 1.326, 1.345, 1.364, 1.383, 1.401, 1.420, 1.439, 1.458, 1.477, 1.496, 1.515, 1.533, 1.552, 1.571, 1.590, 1.609, 1.628])
+        pw.bins = [[w-0.0095e-6, w+0.0095e-6] for w in pw.wavelengths]
+        pw.depths = 1e-6 * np.array([14512.7, 14546.5, 14566.3, 14523.1, 14528.7, 14549.9, 14571.8, 14538.6, 14522.2, 14538.4, 14535.9, 14604.5, 14685.0, 14779.0, 14752.1, 14788.8, 14705.2, 14701.7, 14677.7, 14695.1, 14722.3, 14641.4, 14676.8, 14666.2, 14642.5, 14594.1, 14530.1, 14642.1])
+        pw.errors = 1e-6 * np.array([50.6, 35.5, 35.2, 34.6, 34.1, 33.7, 33.5, 33.6, 33.8, 33.7, 33.4, 33.4, 33.5, 33.9, 34.4, 34.5, 34.7, 35.0, 35.4, 35.9, 36.4, 36.6, 37.1, 37.8, 38.6, 39.2, 39.9, 40.8])
```

Comment:
@julesfowler Good question.  I more or less pulled this directly from the `platon` demos.  This also exists in the demo notebook, but I thought it would be good to also have an example within the module itself.